2024/cfp-main-2024.md

```diff
@@ -0,0 +1,293 @@
+   local Python community.
+3. Short biography of the key members and their previous experience in
+   conference and events organizations.
+4. Previous conference history of the on-site team (if any). Describe which
+   conferences were run and provide some figures on their size (registrations,
+   tracks, total income)
+5. Assignment of key positions to on-site team members. Positions to be assigned
+   (one member can be responsible for more than one of the following roles):
+   1. Chairperson of the on-site team workgroup: usually also the conference chair and representative of the
+      conference for that year
+   2. Sponsor manager: on-site contact for all sponsor activities, responsible for local sponsors
+   3. Venue manager: handling all contacts with the venue and caterer
+   4. Logistics manager: handling all aspects of local logistics and customs
+   5. Conference desk manager: responsible for the on-site help desk, on-site registrations, tickets, cash refunds,
+      attendee support, etc.
+6. List of all team members that will be present during the conference itself as
+   volunteers (but not necessarily help before the conference begins), and/or sources from which people can be or
+   will be acquired if needed (e.g., universities, other local associations, etc.). This list is indicative,
+   as most of the volunteers helping during the conference are usually enrolled during the last months/weeks before
+   the conference.
+7. List of potential international and local sponsors. Note : Local sponsors help a
+   lot in funding the conference. These are often smaller companies which will not
+   move to new locations with the conference, so a local contact is highly
+   beneficial in attracting these sponsors.
+   1. The PAC can assist with contacts with international sponsors where possible
+8. Proposed dates for PyCon APAC 2024.
+9. Keynote speakers proposed.
+10. Conference events. Describe which kind of (social) events you can propose for the conference and are able to
+    provide support for.
+11. Expected differences from previous PyCon APAC conferences. Please highlight what things would need to be changed
+    in the way the conference is run on-site. Examples of things we would consider worthwhile to mention:
+    1. "We can get additional support from the local government / city council"
+    2. "We need to provide tickets to city locals at reduced prices"
+    3. "We intend to serve meals in a different venue"
+    4. "We have some extra space available for posters / sponsors"
+    5. "We want to have our local Python conference run as satellite event" etc.
+12. Venue proposal. Describe the following sub-items:
+    1. Location, reachability by car, train, plane
+    2. Venue floor plans, catering floor plans (if different from venue)
+    3. Total capacity for talk rooms, training rooms, plenary sessions, booth
+       space and catering/restaurants; if possible, including room plans
```

Comment:
We should also ask for individual capacities of broken down rooms (or maybe this is the intended meaning?)

Reply:
I feel like this means "as much detail as possible". We could rephrase this like this:

Provide detailed information on the number of people you can accommodate in lecture rooms, training spaces, main session halls, exhibition areas, and dining spots. If you can, include diagrams or layouts of these spaces.